sleap/io/cameras.py

```diff
@@ -1676,19 +1746,17 @@ def numpy(
                         f"{self.instance_groups}"
                     )
 
-        instance_group_numpys: List[np.ndarray] = []  # len(T) M=all x N x 2
+        instance_group_numpys: List[np.ndarray] = []  # len(T) M=include x N x 2
```

Comment:
is the comment right here?

Reply:
Yes, in the for loop below, we add `T` elements to the `instance_group_numpys` list each with shape `M=include x N x 2`